bower.json

```diff
@@ -36,12 +36,11 @@
     "jquery-colorbox": "1.4.36",
     "jquery-ui": "1.10.3",
     "jquery.cookie": "1.4.0",
-    "js-emoji": "snicker/js-emoji#f25d8a303f",
-    "remarkable": "^1.6.2",
     "ngInfiniteScroll": "1.0.0",
     "pnotify": "1.3.1",
     "sticky": "*",
-    "swagger-ui": "wordnik/swagger-ui#v2.0.24"
+    "swagger-ui": "wordnik/swagger-ui#v2.0.24",
+    "habitica-markdown": "habitrpg/habitica-markdown#emoji"
```

Comment:
This should be switched to a version number once the emoji branch is merged into master.